components/motor/dmc4000/dmc.go

```diff
@@ -497,10 +497,10 @@ func (m *Motor) stopJog() error {
 func (m *Motor) GoFor(ctx context.Context, rpm, revolutions float64, extra map[string]interface{}) error {
 	switch speed := math.Abs(rpm); {
 	case speed < 0.1:
-		m.c.logger.Warnf("motor (%s) speed is nearly 0 rev_per_min", m.Name())
+		m.c.logger.Warn("motor speed is nearly 0 rev_per_min")
 		return motor.NewZeroRPMError()
-	case speed > m.MaxRPM-0.1:
-		m.c.logger.Warnf("motor (%s) speed is nearly the max rev_per_min (%f)", m.Name(), m.MaxRPM)
+	case m.MaxRPM > 0 && speed > m.MaxRPM-0.1:
```

Comment:
[minor] cna you max any public `MaxRPM` struct values private (`maxRPM`). Don't bother with other changes for this PR.

Reply:
in the struct -> change to private. Hopefully no private and public MaxRPM fields exist anywhere.

Reply:
I think there was one motor that had both but I might've been looking at the config and the motor struct. I'll double check!

Reply:
@randhid what about fake/motor.go? should that maxrpm be public still or private as well?